assets/app/view/game/spreadsheet.rb

```diff
@@ -36,7 +36,7 @@ def render_table
           h(:thead, [
             h(:tr, [
               h(:th, ''),
-              h(:th, { attrs: { colspan: @game.players.size } }, 'Player Finances'),
+              h(:th, { attrs: { colspan: @game.all_players.size } }, 'Player Finances'),
```

Comment:
if a player is bankrupt can tehy come back? do we need to show them anywhere?

Reply:
They need to be shown in the endgame. In some games (1849) they can come back.

The spreadsheet view feels incomplete to me if bankrupt players are just entirely omitted, but I suppose closed corporations are omitted. I'm happy to remove them from the sheet if that makes more sense.

Reply:
the only reason to omit would be if we can get rid of the all_players method. that way it's a bit simpler.

if they come back though, that means they are in players array.

Reply:
69ce488d

Need to reject bankrupt players in a few places but it's not too bad.